### PR TITLE
Update frameworks

### PR DESCRIPTION
**Pull request by samuelhwilliams**
## Summary
Updates for search filters.

### CODE REVIEW

File: frameworks/g-cloud-9/questions/services/dataProtectionBetweenNetworks.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ options:
     filter_label: bonded fibre optic connections
   - label: Legacy SSL and TLS (under 1.2)
     value: legacy_ssl
+    filter_ignore: true
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/dataProtectionBetweenNetworks.yml]: So we don't let people select insecure stuff - nice :)

File: frameworks/g-cloud-9/questions/services/multiqPublicSectorNetworks.yml [hunk 2]
```diff
@@ -4,7 +4,6 @@ question: Public sector networks
 depends:
   - "on": lot
     being:
-      - cloud-hardware
```
Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/multiqPublicSectorNetworks.yml]: Wat?!  _checks Git blame_ 😛

Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/multiqPublicSectorNetworks.yml]: _Checks who reviewed the PR_ :P

Comment (TheDoubleK) [inline frameworks/g-cloud-9/questions/services/multiqPublicSectorNetworks.yml]: Ha! I remember it well, was that the one with about 250 updated files?

Comment (samuelhwilliams) [inline frameworks/g-cloud-9/questions/services/multiqPublicSectorNetworks.yml]: Yeah. 😆

**Pull request merged**